tests/unittests/OperatorTest.cpp

```diff
@@ -2367,6 +2367,22 @@ TEST_P(Operator, NonSquarePaddingMaxPool) {
   EXPECT_TRUE(result.isEqual(result1));
 }
 
+TEST_P(Operator, Int8Poolavg) {
+  auto *input =
+      mod_.createVariable(ElemKind::Int8QTy, {1, 3, 3, 1}, 1, 0, "input");
+  input->getHandle<int8_t>() = {0, 1, 2, 3, 4, 5, 6, 7, 8};
+  auto *Pool = F_->createAvgPool("pool", input, 2, 1, {0, 0, 0, 0});
+  auto *S = F_->createSave("save", Pool);
+  EE_.compile(CompilationMode::Infer, F_);
+  EE_.run({}, {});
+  auto result = S->getVariable()->getHandle<int8_t>();
+  auto *output = mod_.createVariable(ElemKind::Int8QTy, {2, 2}, 1, 0, "output");
```

Comment:
why do you need to create a var for this?

Reply:
changed to use a tensor